packages/icons/src/svg/placeholder.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50">

```

Comment:
this seam to be overly complex for a simple icon, maybe path should be optimized a bit ?